src/main/java/com/searchservice/app/rest/ManageTableResource.java

```diff
@@ -118,21 +117,21 @@ public ResponseEntity<Response> createTable(@RequestParam int tenantId, @Request
 			log.error("Table Name  {} is Invalid", manageTableDTO.getTableName());
 
 			throw new InvalidInputOccurredException(HttpStatusCode.INVALID_TABLE_NAME.getCode(),
-					"Creating Table Failed , as Invalid Table Name " + manageTableDTO.getTableName() + " is Provided");
+					"Creating Table Failed ,"+HttpStatusCode.INVALID_TABLE_NAME.getMessage()+" " + manageTableDTO.getTableName() + " is Provided");
 		} else {
 			if (tableDeleteServicePort.isTableUnderDeletion(manageTableDTO.getTableName())) {
-				throw new BadRequestOccurredException(400,
-						"Table With Same Name " + manageTableDTO.getTableName() + " is Marked For Deletion");
+				throw new DeletionOccurredException(HttpStatusCode.UNDER_DELETION_PROCESS.getCode(),
+						"Table With Same Name " + manageTableDTO.getTableName() + MSG_SEPERATOR+HttpStatusCode.UNDER_DELETION_PROCESS.getMessage());
 			} else {
 				manageTableDTO.setTableName(manageTableDTO.getTableName() + "_" + tenantId);
 				Response apiResponseDTO = manageTableServicePort.createTableIfNotPresent(manageTableDTO);
  
 				if (apiResponseDTO.getStatusCode() == 200) {
 					apiResponseDTO.setMessage(
-							"Table-" + manageTableDTO.getTableName().split("_")[0] + ", is created successfully");
+							TABLE + manageTableDTO.getTableName().split("_")[0] + ", is created successfully");
 					return ResponseEntity.status(HttpStatus.OK).body(apiResponseDTO);
 				} else {
-					log.info("Table could not be created: {}", apiResponseDTO);
+					log.info(TABLE +"could not be created: {}", apiResponseDTO);
 					throw new BadRequestOccurredException(101, "REST operation could not be performed");
```

Comment:
Take the StatusCode & Message from HttpStatusCode here

Reply:
Here, you have taken code that is provided in HttpStatusCode, which doesn't satisfy its application as per the corressponding response message given for that, fix that please